distributed/bokeh/worker_monitor.py

```diff
@@ -59,6 +67,11 @@ def resource_append(lists, msg):
     for k in ['cpu', 'memory-percent']:
         lists[k].append(mean(pluck(k, L)) / 100)
 
+    try:
+        net = mean(pluck('network-send', L)) if list(pluck(k, L)) else 0
+    except KeyError:
```

Comment:
This was required because network-send was not included in some messages (particularly the first message at worker startup)